pkg/testgridanalysis/testidentification/ocp_variants.go

```diff
@@ -141,6 +143,9 @@ func (v openshiftVariants) IdentifyVariants(jobName string) []string {
 	if azureRegex.MatchString(jobName) {
 		variants = append(variants, "azure")
 	}
+	if compactRegex.MatchString(jobName) {
+		variants = append(variants, "compact")
+	}
```

Comment:
I *think* we want this below the promote variant and above aws around line 137.

The variants get considered in a sorted order, so I think AWS or azure compact jobs wouldn't be put in the compact bucket

Reply:
It seems to bucket them correctly as-is:
![Screen Shot 2021-07-07 at 9 05 26 AM](https://user-images.githubusercontent.com/1221263/124764026-8c32cc00-df02-11eb-918b-3b6e05b3828e.png)

Though I am open to suggestion.

Reply:
Oh ok, sorry - you're right. Promote returns early, which is why those jobs get excluded from other buckets.

Reply:
this should be fine, jobs can go into multiple variant buckets, with a few exceptions (the cases where we return early: `never-stable` and `promote`)